packages/pharos-cli/src/utils/generation-utils.ts

```diff
@@ -68,3 +68,29 @@ export const addComponentToIndexFile = (
 
   console.log(colors.green(`\nPharos${nameOptions.titleCaseName} export added to ${filePath}`));
 };
+
+export const addComponentsToInitFile = (
+  currentDirectory: FilePath,
+  nameOptions: ComponentNameOptions
+): void => {
+  const componentName = `Pharos${nameOptions.titleCaseName}`;
+  const unitTestsFilePath: FilePath = `${currentDirectory}/packages/pharos/src/test/initComponents.ts`;
+  const storybookFilePath: FilePath = `${currentDirectory}/.storybook/initComponents.js`;
+
+  // import new component in list of imports
+  // and then register new component
+  [unitTestsFilePath, storybookFilePath].map((filePath: FilePath) => {
```

Comment:
Probably better to use `.forEach` here, as you aren't using the returned array from `.map`.

Reply:
switched to `forEach`. So resolving the conversation.